county_criminal_search.go

```diff
@@ -0,0 +1,125 @@
+		FullName          string      `json:"full_name"`
+		Dob               string      `json:"dob"`
+		Yob               int         `json:"yob"`
+		County            string      `json:"county"`
+		State             string      `json:"state"`
+		Address           struct {
+			Street  string `json:"street"`
+			Unit    int    `json:"unit"`
+			City    string `json:"city"`
+			State   string `json:"state"`
+			Zipcode string `json:"zipcode"`
+			Country string `json:"country"`
+		} `json:"address"`
+		Charges []struct {
+			Charge          string      `json:"charge"`
+			ChargeType      interface{} `json:"charge_type"`
+			ChargeID        interface{} `json:"charge_id"`
+			Classification  string      `json:"classification"`
+			Deposition      interface{} `json:"deposition"`
+			Defendant       string      `json:"defendant"`
+			Plaintiff       interface{} `json:"plaintiff"`
+			Sentence        string      `json:"sentence"`
+			Disposition     string      `json:"disposition"`
+			ProbationStatus interface{} `json:"probation_status"`
+			OffenseDate     string      `json:"offense_date"`
+			DepositionDate  string      `json:"deposition_date"`
+			ArrestDate      string      `json:"arrest_date"`
+			ChargeDate      interface{} `json:"charge_date"`
+			SentenceDate    string      `json:"sentence_date"`
+			DispositionDate string      `json:"disposition_date"`
+			ConvictionDate  string      `json:"conviction_date"`
+			ReleaseDate     string      `json:"release_date"`
+			NextCourtDate   string      `json:"next_court_date"`
+			Court           string      `json:"court"`
+			Plea            interface{} `json:"plea"`
+			Assessment      interface{} `json:"assessment"`
+			PrisonTime      string      `json:"prison_time"`
+			JailTime        string      `json:"jail_time"`
+			ProbationTime   interface{} `json:"probation_time"`
+			Restitution     string      `json:"restitution"`
+		} `json:"charges"`
+	} `json:"records"`
+	FilteredByPositiveAdjudicationRecords []struct {
+		ID                string      `json:"id"`
+		CaseNumber        string      `json:"case_number"`
+		FileDate          interface{} `json:"file_date"`
+		ArrestingAgency   string      `json:"arresting_agency"`
+		CourtJurisdiction interface{} `json:"court_jurisdiction"`
+		CourtOfRecord     interface{} `json:"court_of_record"`
+		FullName          string      `json:"full_name"`
+		Dob               string      `json:"dob"`
+		Yob               int         `json:"yob"`
+		County            string      `json:"county"`
+		State             string      `json:"state"`
+		Address           struct {
+			Street  string `json:"street"`
+			Unit    int    `json:"unit"`
+			City    string `json:"city"`
+			State   string `json:"state"`
+			Zipcode string `json:"zipcode"`
+			Country string `json:"country"`
+		} `json:"address"`
+		FilteredByPositiveAdjudicationCharges []struct {
+			Charge          string      `json:"charge"`
+			ChargeType      interface{} `json:"charge_type"`
```

Comment:
@chandradeepak did you use [this](https://mholt.github.io/json-to-go/)? Because I'm seeing a lot of `interfaces` and that's not ideal given that most of the `null` are strings.

Reply:
@magicalbanana  yeah i will do that change for all of them. should i just replace them with string then ?

Reply:
@chandradeepak it depends on what the actual value is. I reached out to Checkr already asking them for a full JSON schema. I think it's safe to say that it will be a string but you might need to do some prior manipulation to the JSON payload to remove the null attributes so they don't get unmarshaled incorrectly.

Reply:
@magicalbanana  fixed it. can you check once.  i want to understand how we are going to use this reports and where